C/CeresSolver/build_tarballs.jl

```diff
@@ -16,8 +16,8 @@ CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix}
               -DBUILD_SHARED_LIBS=ON
               -DBUILD_EXAMPLES=OFF
               -DBUILD_TESTING=OFF
-              -DBLAS_LIBRARIES=${libdir}/libopenblas.${dlext}
-              -DLAPACK_LIBRARIES=${libdir}/libopenblas.${dlext}
+              -DBLAS_LIBRARIES=${libdir}/libblastrampoline.${dlext}
+              -DLAPACK_LIBRARIES=${libdir}/libblastrampoline.${dlext}
```

Comment:
With #8414 this should be simpler, but I'm not sure about what we have to do exactly.  @imciner2?

Reply:
To use that, we need a dependency on libblastrampoline that is new enough, a HostBuildDependency on CMake (either unversioned or at least 3.24), do `apk del cmake` in the script, add `-DCMAKE_PROJECT_TOP_LEVEL_INCLUDES="${libdir}/cmake/blastrampoline/yggdrasilenv.cmake"` to the CMake options, and remove these existing options.